README.md

```diff
@@ -63,3 +63,14 @@ If you want to continuously update your running behaviors while in edit mode (as
 Sometimes it is necessary for Unity systems to add hidden objects to the user's scene. Either the object should not be selected and modified, should not be included in Player builds, or needs to be hidden for otherr reasons.
 
 The HiddenHierarchy window shows a Hierarchy-like view of the currently open scenes which includes all GameObjects, even those normally hidden in the hierarchy. This is useful for debugging systems involving hidden objects, in case new objects "leak" into the scene or the system somehow fails to destroy a hidden object.
+
+## ModificationResponse
+This is an example of how to hook into Undo.postprocessModifications and Undo.undoRedoPerformed to respond to property modifications in a Scene.  It uses a short timer that is reset and started when a change is detected, and it only triggers the response when the timer finishes.  This pattern is useful when you have a complex response that you don't want to happen constantly as a continuous property is changed (for example, as a user drags a slider in the Inspector).
+
+## AssetsInSync
+This is an example of how to use the OnWillSaveAssets callback in AssetModificationProcessor to keep two Assets in sync.
```

Comment:
I think I'm going to change this example to be specifically about keeping a SO in sync with a scene, since this comes from our use case of storing scene metadata.